src/lug/Window/Unix/WindowImplX11.cpp

```diff
@@ -0,0 +1,16 @@
+#include <lug/Window/Unix/WindowImplX11.hpp>
+
+lug::Window::priv::WindowImpl::WindowImpl()
+{
```

Comment:
`() {}`

same remark for other places in this PR, but will not comment each time